parm/stage/fv3_cold.yaml.j2

```diff
@@ -0,0 +1,11 @@
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+fv3_cold:
+    mkdir:
+        - "{{ COM_ATMOS_INPUT }}"
+    copy:
+        - ["{{ BASE_CPLIC }}/{{ CPL_ATMIC }}/{{ cycle_YMDH }}/{{ MEMDIR }}/atmos/gfs_ctrl.nc", "{{ COM_ATMOS_INPUT }}"]
+        {% for ftype in ["gfs_data", "sfc_data"] %}
+        {% for ntile in range(1, ntiles + 1) %}
+        - ["{{ BASE_CPLIC }}/{{ CPL_ATMIC }}/{{ cycle_YMDH }}/{{ MEMDIR }}/atmos/{{ ftype }}.tile{{ ntile }}.nc", "{{ COM_ATMOS_INPUT }}"]
+        {% endfor %} # ntile
+        {% endfor %} # ftype
```

Comment:
You have the option to tab Jinja blocks.  The YAML still has to be tabbed correctly, though.  So you could do
```jinja,yaml
        {% for ftype in ["gfs_data", "sfc_data"] %}
            {% for ntile in range(1, ntiles + 1) %}
        - ["{{ BASE_CPLIC }}/{{ CPL_ATMIC }}/{{ cycle_YMDH }}/{{ MEMDIR }}/atmos/{{ ftype }}.tile{{ ntile }}.nc", "{{ COM_ATMOS_INPUT }}"]
            {% endfor %} # ntile
        {% endfor %} # ftype
```

or
```jinja
{% for ftype in ["gfs_data", "sfc_data"] %}
    {% for ntile in range(1, ntiles + 1) %}
        - ["{{ BASE_CPLIC }}/{{ CPL_ATMIC }}/{{ cycle_YMDH }}/{{ MEMDIR }}/atmos/{{ ftype }}.tile{{ ntile }}.nc", "{{ COM_ATMOS_INPUT }}"]
    {% endfor %} # ntile
{% endfor %} # ftype
```
The latter option doesn't always work out so nicely

Reply:
Good to know, thanks! I'll indent those secondary `for` loops to make it more readable in the yamls. Done @ 3f08f035